packages/edit-site/src/components/template-details/index.js

```diff
@@ -49,17 +55,18 @@ export default function TemplateDetails( { template, onClose } ) {
 
 	return (
 		<div className="edit-site-template-details">
-			<div className="edit-site-template-details__group">
+			<VStack className="edit-site-template-details__group" spacing={ 3 }>
 				{ canEditTitle ? (
 					<EditTemplateTitle template={ template } />
 				) : (
-					<Heading
-						level={ 4 }
+					<Text
+						size={ 16 }
```

Comment:
I ported this from https://github.com/WordPress/gutenberg/pull/42495. Props to @afercia.